Tests/DependencyInjection/MonologExtensionTest.php

```diff
@@ -521,11 +525,49 @@ public function testFingersCrossedHandlerWhenExcluded404sAreSpecified()
         $this->assertDICConstructorArguments($handler, [new Reference('monolog.handler.nested'), new Reference('monolog.handler.main.not_found_strategy'), 0, true, true, null]);
     }
 
-    public function testFingersCrossedHandlerWhenExcludedHttpCodesAreSpecified()
+    /** @group legacy */
```

Comment:
what is legacy in this test?

Reply:
this was already marked as legacy before, he's just dupplicated it to add a Symfony 5.2+ version.